app/app/explore/components/AddLiquidityModal.tsx

```diff
@@ -60,105 +55,31 @@ export default function AddLiquidityModal({
   const [assetAmount, setAssetAmount] = useState("");
   const [runeAmount, setRuneAmount] = useState("");
   const [txHash, setTxHash] = useState<string | null>(null);
-  const [assetBalance, setAssetBalance] = useState(0);
-  const [balanceLoading, setBalanceLoading] = useState(false);
   const [showConfirmation, setShowConfirmation] = useState(false);
   const [isSubmitting, setIsSubmitting] = useState(false);
   const [isDualSided, setIsDualSided] = useState(false);
-  const {
-    runeBalance,
-    loading: runeBalanceLoading,
-    error: runeBalanceError,
-  } = useRuneBalance({ wallet: selectedWallet });
 
   const { positions, markPositionAsPending } = useLiquidityPositions();
 
-  const utxoChain = useMemo(() => {
-    const chain = pool.asset.split(".")[0].toLowerCase();
-    if (chain === "btc") return "BTC";
-    if (chain === "doge") return "DOGE";
-    return null;
-  }, [pool.asset]);
-
   const poolNativeDecimal = parseInt(pool.nativeDecimal);
   const assetMinimalUnit = 1 / 10 ** poolNativeDecimal;
   const runeMinimalUnit = 1 / 10 ** DECIMALS;
+  const runeBalance = balanceList[ChainKey.THORCHAIN]["THOR.RUNE"]
+    ? balanceList[ChainKey.THORCHAIN]["THOR.RUNE"].balance
+    : 0;
+  const assetBalance =
+    balanceList[
+      getChainKeyFromChain(assetFromString(pool.asset)?.chain as string)
+    ][pool.asset].balance;
 
   if (!selectedWallet?.provider) {
     throw new Error("Wallet provider not found, please connect your wallet.");
   }
 
-  const {
-    getBalance: getUTXOBalance,
-    loading: utxoLoading,
-    error: utxoError,
-  } = useUTXO({
-    chain: utxoChain as "BTC" | "DOGE",
```

Comment:
The removed code in this file is relied upon by the work that @desamtralized did in https://github.com/yieldi-labs/yieldi-web/pull/52

@hippocampus-web3 can you rebase your branch with @desamtralized to make sure that this PR does not impact his work?